packages/databricks-vscode/src/bundle/GenerateBundle.ts

```diff
@@ -1,16 +1,64 @@
 import {CliWrapper} from "../cli/CliWrapper";
 import {extensions, Uri} from "vscode";
-import path from "node:path";
+import {workspace, RelativePattern, GlobPattern, Disposable} from "vscode";
+import YAML from "yaml";
 
-export async function generateBundleSchema(cli: CliWrapper) {
+export async function generateBundleSchema(
+    cli: CliWrapper
+): Promise<Disposable> {
     // get freshly generated bundle schema
     const bundleSchema = await cli.getBundleSchema();
 
     // URI scheme for DABs JSON schemas
     const dabsUriScheme = "dabs";
 
     // URI for bundle root config json schema
-    const rootConfigSchemaUri = `${dabsUriScheme}:///root.json`;
+    const rootConfigSchemaUri = `${dabsUriScheme}:///dabs.json`;
+
+    const folder = workspace.workspaceFolders?.[0];
+    const configFilePattern = new RelativePattern(
+        folder!,
+        "{databricks,bundle}.{yml,yaml}"
+    );
+
+    // file watcher on all YAML files
+    const watcher = workspace.createFileSystemWatcher("**/*.{yml,yaml}");
+    watcher.onDidChange(async () => {
```

Comment:
Let's use onDidCreate and onDidDelete

Reply:
We also need a watcher on root bundle.yaml, so that we can update globs when the root is updated. 
IMO we ONLY need the root watcher.

Reply:
We need an `onDidCreate` watcher for `*.yml` and an `onDidChange` watcher for `{databricks|bundle}.{yaml|yml}`.

That would be a bit more efficient but I feel that this code is easier to read.

We need the watcher on z`onDidCreate` in order to cover include patterns like `*.yml`.